live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/messaging/in/RepoEventMapperTest.java

```diff
@@ -49,10 +50,7 @@
 import org.alfresco.hxi_connector.live_ingester.domain.usecase.metadata.IngestMetadataCommand;
 import org.alfresco.hxi_connector.live_ingester.domain.usecase.metadata.model.CustomPropertyDelta;
 import org.alfresco.hxi_connector.live_ingester.messaging.in.mapper.RepoEventMapper;
-import org.alfresco.repo.event.v1.model.DataAttributes;
-import org.alfresco.repo.event.v1.model.NodeResource;
-import org.alfresco.repo.event.v1.model.RepoEvent;
-import org.alfresco.repo.event.v1.model.UserInfo;
+import org.alfresco.repo.event.v1.model.*;
 
 class RepoEventMapperTest
```

Comment:
I don't like this test at all but i don't see any easy way to improve it. Do you have any idea @krdabrowski, @mpichura?